.github/workflows/condarise.yaml

```diff
@@ -2,8 +2,8 @@ name: Condarise
 on: [push, pull_request]
 jobs:
   build-publish:
-    # Run for tags only
-    if: "startsWith(github.ref, 'refs/tags/')"
+    # Run on merge to master, where the commit name starts with "Bump version:" (for bump2version)
+    if: "startsWith(github.ref, 'refs/heads/main') && startsWith(github.event.head_commit.message, 'Bump version:')"
```

Comment:
I'd use:

```
on:
  push:
    branches:
      - main
```

Instead of `startsWith(github.ref, 'refs/heads/main')` (like here https://github.com/populationgenomics/hail/blob/main/.github/workflows/condarise.yaml#L2-L5)

But probably shouldn't matter. 

Except that I'd remove `pull_request` anyway avoid building packages on unmerged PRs. What do you think?

Reply:
Yeah totally agree, updated :)